client/me/billing-history/receipt.jsx

```diff
@@ -126,6 +130,36 @@ const BillingReceipt = React.createClass( {
 		);
 	},
 
+	renderTitle() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<QueryBillingTransactions />
```

Comment:
Looks better this way, but I'll leave the refactor for a next PR. Adding to #10554.